src/ui/ConsoleView.js

```diff
@@ -14,6 +15,8 @@ class ConsoleView extends Component {
       path: settings.lastConsolePath || '/config',
       body: null,
       json: null,
+      loading: false,
```

Comment:
loading and loaded are mutually exclusive. This better to be enum instead of two booleans.

Reply:
Good point. Let me fix this.